src/components/dapps/ModalCreateDapps.vue

```diff
@@ -388,11 +239,14 @@ export default defineComponent({
 
     const openOption = ref(false);
 
-    const unit_d = 3;
-    const decimal = 12;
+    const unitToken = inject('unitToken', '');
+    const decimal = inject('decimal', 10);
+
+    const selectUnitEndowment = ref(unitToken);
+    const selectUnitGas = ref(unitToken);
 
     const formData = reactive<FormData>({
-      endowment: bnToBn(plasmUtils.reduceDenomToBalance(27, unit_d, decimal)),
+      endowment: bnToBn(plasmUtils.reduceDenomToBalance(27, 3, decimal)),
```

Comment:
The input form shows `27^15` in the default token denominator. The value should be converter from femto instead of UNIT.
![bandicam 2021-07-05 18-06-37-170](https://user-images.githubusercontent.com/40356749/124448001-eb86c400-ddbc-11eb-8ffb-86ed38ac7f8d.jpg)

Reply:
Fixed for converter to other unit